### ISSUE TITLE
music playlist crashing xbmc

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#193]
```
What steps will reproduce the problem?
1. play music on xbmc
2. go to music playlist on xbmc remote
3. select a different song
4. xbmc remote crashes

What is the expected output? What do you see instead?

different song in my ear...crash and burn of xbmc remote

What version of XBMC Remote are you using?

XBMC_Remote_v0.6.6_signed.apk 

Which Android device are you using (and which ROM, if custom)?

droid 2.0

Which XBMC revision are you running and on which platform?

camelot 9.11


Please provide any additional information below.

```

Original issue reported on code.google.com by `mpinte...@gmail.com` on 12 Mar 2010 at 1:30

**Comment by GoogleCodeExporter** [comment]
```
by xbmc i mean xbmcremote.  sorry for the confusion.
```

Original comment by `mpinte...@gmail.com` on 12 Mar 2010 at 1:30

**Comment by GoogleCodeExporter** [comment]
```
I tested that quite well actually. Will try to reproduce with a Droid similar 
emulator.
```

Original comment by `phree...@gmail.com` on 25 Mar 2010 at 11:57
- Changed state: **Accepted**
- Removed labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
Can't reproduce this with Nexus 2.1 update 1. Guess that's the famous 2.0 bug...
```

Original comment by `till.ess...@googlemail.com` on 8 Apr 2010 at 7:01

**Comment by GoogleCodeExporter** [comment]
```
closing. all that stuff should be fixed in 0.6.7. reopen if that's not the case.
```

Original comment by `phree...@gmail.com` on 20 Apr 2010 at 11:30
- Changed state: **Fixed**
